src/devices/Ads1115/InputMultiplexeConfig.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ads1115
+{
+    /// <summary>
+    ///  Configure the Input Multiplexer
+    /// </summary>
+    public enum InputMultiplexeConfig
+    {
+        /// <summary>
+        ///  AINP = AIN0 and AINN = GND 
+        /// </summary>
```

Comment:
It would help if you provide somewhere in this file (perhaps on the InputMultiplexerConfig comment) what does AINP and AINN mean.